acrobat/scripts/scripts.js

```diff
@@ -435,6 +435,11 @@ const { ietf } = getLocale(locales);
   loadIms().then(() => {
     const imsIsReady = new CustomEvent('IMS:Ready');
     window.dispatchEvent(imsIsReady);
+    if (!window.adobeIMS.isSignedInUser()) {
+      if (widgetBlock) {
```

Comment:
There are two conditions. Which one goes first is more efficient? How about
```
  if (widgetBlock && !window.adobeIMS.isSignedInUser()) {
```

Reply:
Let's do it